cmd/describe/upgrade/cmd.go

```diff
@@ -59,18 +59,17 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 	if len(upgrades) < 1 {
-		r.Reporter.Errorf("No available upgrades for cluster id '%s'", cluster.ID())
+		r.Reporter.Warnf("No scheduled upgrades for cluster id '%s'", cluster.ID())
 		os.Exit(1)
 	}
 
 	for _, upgrade := range upgrades {
-		fmt.Printf(`
-                %-28s%s
+		fmt.Printf(`%-44s%s
 		%-28s%s
 		%-28s%s
                 %-28s%s
 `,
-			"ID:", upgrade.ID(),
+			"                ID:", upgrade.ID(),
```

Comment:
Is it possible to use padding-left instead?
https://golang.cafe/blog/golang-string-padding-example.html

Reply:
That will solve it https://go.dev/play/p/V9XtXkGBYP1, WDYT?
I added padding to the left.

Reply:
that's great

Reply:
```
~/workPlace/rosa (describe-ux-improvement)! S $ ./rosa describe upgrade -c 20s72fgq7lc5ga95hci1b48i6p53443q
                ID:3dd700a1-85cc-11ed-9508-0a580a830646
                                Cluster ID:                 20s72fgq7lc5ga95hci1b48i6p53443q
                                Next Run:                   2022-12-27 10:02:00 +0000 UTC
                Upgrade State:              scheduled
Version: 4.10.45            
```

it doesn't work for me. does it look good on your end?

Reply:
I've tested it in the go playground with dummy values, can you share the code changes?

Reply:
Ok, I played a bit with the values in the ROSA CLI:
![image](https://user-images.githubusercontent.com/57869309/209672694-2e6f58d5-5685-46f0-b610-b2aace5d96c0.png)

Reply:
changed it to :
```
for _, upgrade := range upgrades {
		fmt.Printf(`%-19s%61s
		%-28s%s
		%-28s%s
		%-28s%s
`,
			"ID:", upgrade.ID(),
			"Cluster ID:", upgrade.ClusterID(),
			"Next Run:", upgrade.NextRun(),
			"Upgrade State:", upgradeState.Value())
```
as you suggested, but it still doesn't work:
```
~/workPlace/rosa (describe-ux-improvement)! S $ ./rosa describe upgrade -c 20s72fgq7lc5ga95hci1b48i6p53443q
ID:                                         7b1db0fe-85ea-11ed-9508-0a580a830646
                Cluster ID:                 20s72fgq7lc5ga95hci1b48i6p53443q
                Next Run:                   2022-12-27 13:39:00 +0000 UTC
                Upgrade State:              scheduled
                Version:                    4.10.45

```

Reply:
Maybe leaving it as I did and saving time would be better? WDYT?

Reply:
Can you please remove the `-`?
We need `fmt.Printf(%19s%61s`, the `-` is adding right padding, while without the dash it adds left padding.

Reply:
NICE
```
~/workPlace/rosa (describe-ux-improvement)! S $ ./rosa describe upgrade -c 20s72fgq7lc5ga95hci1b48i6p53443q
                ID:                         7b1db0fe-85ea-11ed-9508-0a580a830646
                Cluster ID:                 20s72fgq7lc5ga95hci1b48i6p53443q
                Next Run:                   2022-12-27 13:39:00 +0000 UTC
                Upgrade State:              scheduled
                Version:                    4.10.45

```

Reply:
Any reason to use the hardcoded values of 19 and 61?

Reply:
This is the number of spaces required to align the first row with the text

Reply:
Since the values of the first column are constants for this command, I'm satisfied with the current solution for this bug.